Add Delete Saved Address Endpoint  #10

Khilan10 opened the pull request:
- Add mapping /address/{address_id} to delete saved address

- Add named query in Address Entity and Customer Address Entity

Khilan10 committed:
Add Delete Saved Address Endpoint  #10

- Add mapping /address/{address_id} to delete saved address

- Add named query in Address Entity and Customer Address Entity

pavankarthiktanikanti reviewed (changes requested):
- Address shouldn't be deleted if status is archived (active as 0)

- Add one more request mapping to handle empty address id for delete

- Handle empty address id before validating user authentication

- Use a Single query to get customer address instead of two queries

Khilan10 committed:
Update delete address Endpoint so that it can  also archive Address  #10

- Removed named query from Address Entity and updated named query in CustomerAddressEntity

- Update delete Address method in Address Controller to handle archiving of Address
